static/styles/main.css

```diff
@@ -1 +1 @@
nt{text-align:left;width:100%}.content time{margin-left:3px}.content a{text-shadow:0.03em 0 #fff, -0.03em 0 #fff, 0 0.03em #fff, 0 -0.03em #fff, 0.06em 0 #fff, -0.06em 0 #fff, 0.09em 0 #fff, -0.09em 0 #fff, 0.12em 0 #fff, -0.12em 0 #fff, 0.15em 0 #fff, -0.15em 0 #fff, 0.03em 0.075em #fff, -0.03em 0.075em #fff, 0.06em 0.075em #fff, -0.06em 0.075em #fff, 0.09em 0.075em #fff, -0.09em 0.075em #fff, 0.12em 0.075em #fff, -0.12em 0.075em #fff, 0.15em 0.075em #fff, -0.15em 0.075em #fff;background-image:linear-gradient(#DA784D, #DA784D);background-size:1px 2px;background-repeat:repeat-x;background-position:0 95%;text-decoration:none}.content a:hover{color:#DA784D}.highlight{margin:10px 0}.pagination{margin:50px 0 0}.pagination :nth-child(2){float:right}.full{height:100vh;top:0;bottom:0}.about{width:100%;background-color:#F7F7F7;padding-bottom:4rem}.about p{font-size:1.666rem;margin-top:2rem}@media (max-width: 48em){.about p{font-size:1.563rem}}@media (max-width: 25em){.about p{font-size:1.375rem}}.about a{text-shadow:0.03em 0 #fff, -0.03em 0 #fff, 0 0.03em #fff, 0 -0.03em #fff, 0.06em 0 #fff, -0.06em 0 #fff, 0.09em 0 #fff, -0.09em 0 #fff, 0.12em 0 #fff, -0.12em 0 #fff, 0.15em 0 #fff, -0.15em 0 #fff, 0.03em 0.075em #fff, -0.03em 0.075em #fff, 0.06em 0.075em #fff, -0.06em 0.075em #fff, 0.09em 0.075em #fff, -0.09em 0.075em #fff, 0.12em 0.075em #fff, -0.12em 0.075em #fff, 0.15em 0.075em #fff, -0.15em 0.075em #fff;background-image:linear-gradient(#DA784D, #DA784D);background-size:2px 3px;background-repeat:repeat-x;background-position:0 95%;text-decoration:none}.about a:hover{color:#DA784D}.gallery div[class^="col-"],.gallery div[class*=" col-"]{padding:0;position:relative}.gallery div[class^="col-"]:hover .overlay,.gallery div[class*=" col-"]:hover .overlay{opacity:1}.gallery img{padding:6rem}.overlay{cursor:pointer;position:absolute;top:0;bottom:0;left:0;right:0;height:100%;width:100%;opacity:0;transition:.2s ease-out;background-color:#212121}.overlay a{color:#FFF}.overlay a:hover{color:#DA784D}.overlay h2{color:#FFF}.overlay h3{font-family:"Rubik","Poppins","Hel­vetica",Arial,sans-serif}.blog{min-height:100vh}.blog h3{margin:0 0 2rem;font-family:"Rubik","Poppins","Hel­vetica",Arial,sans-serif}.contact{width:100%;background-color:#F7F7F7;padding-bottom:4rem}.contact form{margin-top:2rem}.contact .contact-submit{position:relative}.contact #form-submit{display:block;margin:0 auto;width:50%}.contact #form-thankyou{position:absolute;top:50%;left:50%;transform:translate(-50%, -50%)}input,textarea{background-color:#FFF;border-color:#F7F7F7;border-radius:3px;border-width:1px;border-style:solid;color:#212121;padding:10px;margin:8px 0;width:100%}input[type="submit"],textarea[type="submit"]{background-color:#4E6B6C;color:#FFF;transition:.2s ease-out}input[type="submit"]:hover,textarea[type="submit"]:hover{background-color:#DA784D;color:#FFF;cursor:pointer}textarea{min-height:8rem;overflow:auto}footer{height:12rem;background-color:#212121}footer div,footer a{color:#FFF}
```

Comment:
It looks like some of the styling was removed - I will rerun my Gulp task to compile and minify.

Reply:
Sure, let me know if I need to do something else.
PD: I have used ruby scss to compile main.css file 
`sass --watch main.scss:../main.css --style compressed --sourcemap=none`
Also include dependencies